container/container.bzl

```diff
@@ -20,6 +20,7 @@ load("//container:import.bzl", "container_import")
 load("//container:load.bzl", "container_load")
 load("//container:pull.bzl", "container_pull")
 load("//container:push.bzl", "container_push")
+load("//contrib:test.bzl", "container_test")
```

Comment:
We don't include `contrib` things here.  If it's ready to graduate it should move to `//container`, but I don't think that's the intent of this change?

Reply:
no.. let me just get it to work here. We have a task internally to convert all image_test.py to use container_test
Maybe then we will ready.. Just created one here https://github.com/bazelbuild/rules_docker/issues/256

Reply:
Should this line be dropped?